enhancements/network/ip-interface-selection.yaml

```diff
@@ -0,0 +1,278 @@
+---
+title: ip-interface-selection
+authors:
+  - @cybertron
+reviewers:
+  - @jcaamano
+  - @tsorya
```

Comment:
in case you want/can/need add me as reviewer too :)

Reply:
Done, thanks!